README.md

```diff
@@ -218,23 +218,25 @@ Python and library versions: Python 3.7.3 | Anaconda, numpy 1.16.4, pillow 6.0.0
 
 
 ## Contributing
-1. Clone the repository:
-   ```
-   git clone git@github.com:albu/albumentations.git
-   cd albumentations
-   ```
-2. Install the library in development mode:
-   ```
-   pip install -e .[tests]
-   ```
-3. Run tests:
-   ```
-   pytest
-   ```
-4. Run flake8 to perform PEP8 and PEP257 style checks and to check code for lint errors.
-   ```
-   flake8
-   ```
+
+1. Fork the repository:
+2. Clone it.
+3. Install pre-commit hook:
+```
+pip install pre-commit
+pre-commit install
+```
+4. Modify it.
```

Comment:
modify pre-commit hook? how?

Reply:
I meant, make desired modification in the code. Not in the pre-commit hook.

Rephrased.